.rubocop.yml

```diff
@@ -8,15 +8,22 @@ AllCops:
 Layout/LineLength:
   Max: 120
 Metrics/MethodLength:
-  Max: 20
+  Max: 100
```

Comment:
- [ ] [REQUIRED] Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))
**NB:** If you can't reduce the lines in your methods, you can disable cops to fix errors like this:
```
  # rubocop:disable Metrics/MethodLength
  This is a long line 
  # rubocop:enable Metrics/MethodLength
```